lib/licenses.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const licenseTypes = {
+  'permissive': {
```

Comment:
Could you explain how these were derived from `https://choosealicense.com/appendix/`, and give attribution? I think that content is licensed CC-BY. It would also make it easier to check your work, as you requested!

Reply:
'public-domain' licenses are those that do not require 'license and copyright notice', 'viral' - requires 'disclose source' or 'same license', rest is 'permissive'. Does it answer your question?

Reply:
Yea, great! Could you add this in a comment? I feel like that'll be helpful to others in the future.

Reply:
done